todays-curiosity.js

```diff
@@ -0,0 +1,111 @@
+class todaysCuriosity {
+  constructor(baseImage) {
+    this.baseImage = baseImage;
+
+    this.createCanvases();
+    this.paintInputImage();
+    this.createBrightpixels();
+    this.setDivisions();
+    this.setReductionRatio();
+    this.setOffset();
```

Comment:
The above three are called in the constructor to load up default values, so that if `getReduced. . .` or `getReveresdPixelBlocks` is called without any of these being called it doesn't break and still serves up valid images.